packages/ui/src/components/Catalog/Catalog.models.ts

```diff
@@ -6,7 +6,8 @@ export interface ITile<T = unknown> {
   headerTags?: string[];
   tags: string[];
   version?: string;
-  rawObject: T;
+  /** DEPRECATED: Please relay on name property instead */
```

Comment:
Does the `@deprecated` do something good?
https://stackoverflow.com/questions/60755711/is-it-possible-to-mark-something-as-deprecated-in-typescript

Reply:
Yes, it does, it shows on VSCode in a better way, and some linters also complains, I'll updated it

Reply:
![image](https://github.com/KaotoIO/kaoto-next/assets/16512618/df76ec08-bad4-48dd-8b0a-64810f55377b)